src/service/imageUploadService.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import { ref, uploadBytes, getDownloadURL } from "firebase/storage";
+import { storage } from "../../firebase";
+import { v4 } from "uuid";
+
+const UploadHandler = () => {
+  const [bannerFile, setBannerFile] = useState(null);
+  const [logoFile, setLogoFile] = useState(null);
+  const [uploading, setUploading] = useState(false);
```

Comment:
Ty can we transfer this kung asa ni siya g.gamit? Wala mani siya g.gamit dri nga service man gud. 
and need ta ani nga service kay ang path which is kung asa siya e.save and unsay name sa file. We can refer sa DeleteImageService as a logic.

Reply:
![image](https://github.com/Hitoru141/EsportsWebsite/assets/101559620/93943879-eb05-420a-8cd5-dd160da310e1)

Reply:
![image](https://github.com/Hitoru141/EsportsWebsite/assets/101559620/93569f96-312d-4539-b26a-84b550eaddfc)